crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -23,40 +23,37 @@ const.ERR_LINK_MAP_MAX_ENTRIES_EXCEEDED="number of link map entries exceeds maxi
 # BOOK KEEPING
 # -------------------------------------------------------------------------------------------------
 
-# The memory address at which the transaction vault root is stored.
-const.TX_VAULT_ROOT_PTR=0
-
 # The memory address at which a pointer to the input note being executed is stored.
-const.CURRENT_INPUT_NOTE_PTR=4
+const.CURRENT_INPUT_NOTE_PTR=0
 
 # The memory address at which the number of output notes is stored.
-const.NUM_OUTPUT_NOTES_PTR=8
+const.NUM_OUTPUT_NOTES_PTR=4
 
 # The memory address at which the input vault root is stored.
-const.INPUT_VAULT_ROOT_PTR=12
+const.INPUT_VAULT_ROOT_PTR=8
 
 # The memory address at which the output vault root is stored.
-const.OUTPUT_VAULT_ROOT_PTR=16
+const.OUTPUT_VAULT_ROOT_PTR=12
 
-# The memory address at which the dirty flag of the storage commitment of the native account is 
-# stored. 
-# 
-# This binary flag specifies whether the commitment is outdated: it holds 1 if some changes were 
+# The memory address at which the dirty flag of the storage commitment of the native account is
+# stored.
+#
+# This binary flag specifies whether the commitment is outdated: it holds 1 if some changes were
 # made to the account storage since the last re-computation, and 0 otherwise.
-const.NATIVE_ACCT_STORAGE_COMMITMENT_DIRTY_FLAG_PTR=20
+const.NATIVE_ACCT_STORAGE_COMMITMENT_DIRTY_FLAG_PTR=21
```

Comment:
Why did this become 21? Shouldn't it have been 16?

Reply:
updated, it was previously mis-calc due to rebasing issue